### DISCUSSION TITLE
add option for auto logout for the mobile app

### DISCUSSION THREAD

**Discussion by bloukanov** [community/community#33546] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.80.0 (78250505)
Device: iPhone15,2 (iOS 16.0.1)
</details>

**Comment by eliperkins** [comment]
Thanks for the feedback here, @bloukanov! Can I ask some more questions to help clarify your goals here?

For autologout, are you looking to ensure your GitHub account cannot be compromised if your device is lost after some period of time? Or are there other features that being logged out of the app will help to address?

**Comment by bloukanov** [comment]
Hey yup! just if i leave it open on a table or something, as an added
protection layer. for example, the bank apps log out each time, and Jira
offers an auto-logout option. then logging in with FaceID is really easy.
is it intentionally designed to be always logged in?
